src/avatar.h

```diff
@@ -74,6 +75,11 @@ struct monster_visible_info {
     void remove_npc( npc *n );
 };
 
+struct point_of_interest {
+    tripoint_abs_omt pos = tripoint_abs_omt::invalid;
```

Comment:
I would *consider* using std::optional here instead of tripoint_abs_omt::invalid as a sentinel value.

Reply:
Doing that would propagate to avatar::get_active_mission_target(), which would either just convert it back to a sentinel value (fairly pointless, in my opinion), or result in it being converted to use std::optional, which would affect all of its usages (there are 7 in total, excluding the operation itself).
I haven't looked at that in detail, but don't imagine it would be difficult to do. If you want me to I'll do it (or discover it might not be a good idea, if it gets messy, which isn't likely).

Reply:
Neither of these are requests, just considerations. I'm satisfied you've given it enough thought.